core/templates/dev/head/profile/Profile.js

```diff
@@ -29,22 +31,87 @@ oppia.controller('Profile', [
     $rootScope.loadingMessage = 'Loading';
     $http.get(profileDataUrl).success(function(data) {
       $rootScope.loadingMessage = '';
+      $scope.username = {
+        title: 'Username',
+        value: data.username,
+        helpText: (data.username)
```

Comment:
Why do we need the help text in the first place? It seems a bit odd in the UI -- it gives no additional information.